global-clusters-automation/failover_and_delete_global_cluster.py

```diff
@@ -148,9 +148,9 @@ def delete_global_cluster(global_cluster_id, secondary_cluster_arn, secondary_cl
         if each_secondary_cluster != secondary_cluster_arn:
             print('Removing secondary cluster ', each_secondary_cluster, ' from global cluster ', global_cluster_id)
             remove_from_global_cluster(each_secondary_cluster, global_cluster_id)
-    # Wait until all standalone clusters are promoted
-    print('Waiting till all secondary clusters are removed from global cluster ', global_cluster_id)
-    wait_for_promotion_to_complete(global_cluster_id, secondary_cluster_arn)
+             # Wait until all standalone clusters are promoted
+            print('Waiting till all secondary clusters are removed from global cluster ', global_cluster_id)
```

Comment:
Modify message to be more relevant, code is actually waiting for the secondary promotion isn't it?

Reply:
Code is actually checking for promotion of all secondary clusters to stand alone clusters